gemfiles/rails_5.1.gemfile.lock

```diff
@@ -0,0 +1,211 @@
+PATH
+  remote: ..
+  specs:
+    asyncapi-server (1.1.3)
+      active_model_serializers (~> 0.9.0)
+      api-pagination
+      ar_after_transaction
+      kaminari
+      rails (>= 4)
+      responders
+      sidekiq
+      typhoeus
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    actioncable (5.1.5)
+      actionpack (= 5.1.5)
+      nio4r (~> 2.0)
+      websocket-driver (~> 0.6.1)
+    actionmailer (5.1.5)
+      actionpack (= 5.1.5)
+      actionview (= 5.1.5)
+      activejob (= 5.1.5)
+      mail (~> 2.5, >= 2.5.4)
+      rails-dom-testing (~> 2.0)
+    actionpack (5.1.5)
+      actionview (= 5.1.5)
+      activesupport (= 5.1.5)
+      rack (~> 2.0)
+      rack-test (>= 0.6.3)
+      rails-dom-testing (~> 2.0)
+      rails-html-sanitizer (~> 1.0, >= 1.0.2)
+    actionview (5.1.5)
+      activesupport (= 5.1.5)
+      builder (~> 3.1)
+      erubi (~> 1.4)
+      rails-dom-testing (~> 2.0)
+      rails-html-sanitizer (~> 1.0, >= 1.0.3)
+    active_model_serializers (0.9.7)
+      activemodel (>= 3.2)
+      concurrent-ruby (~> 1.0)
+    activejob (5.1.5)
+      activesupport (= 5.1.5)
+      globalid (>= 0.3.6)
+    activemodel (5.1.5)
+      activesupport (= 5.1.5)
+    activerecord (5.1.5)
+      activemodel (= 5.1.5)
+      activesupport (= 5.1.5)
+      arel (~> 8.0)
+    activesupport (5.1.5)
+      concurrent-ruby (~> 1.0, >= 1.0.2)
+      i18n (~> 0.7)
+      minitest (~> 5.1)
+      tzinfo (~> 1.1)
+    api-pagination (4.7.0)
```

Comment:
On asyncapi-client, I have the following on my changelog:

`include Rails::Paginate for api-pagination 4.5 or higher`

FYI in case you're using this.

Reply:
I have it documented on commit 3a1ab7e3ca6d15bd527dd469e23a1a60a16b720b 

Thanks for the heads up! 👍